src/webgpu/api/validation/createTexture.spec.ts

```diff
@@ -361,6 +460,92 @@ g.test('texture_size,3d_texture')
     }, !success);
   });
 
+g.test('texture_size,3d_texture,compressed_format')
```

Comment:
Not sure whether this is needed. We say that compressed formats (BC formats currently) are not supported in 3D texture in Dawn project: https://dawn-review.googlesource.com/c/dawn/+/34922/1/src/tests/unittests/validation/TextureValidationTests.cpp#561

Reply:
I think it might not be needed because my understanding (based on your comment) is that WebGPU shouldn't enable 3D compressed textures at this point.

Reply:
+1, no 3d compressed textures at this point. I tried to investigate it but got confused by the D3D docs which seem to say BC texture blocks are 4x4x4 but I don't understand how that's possible. It's too niche and too complicated for V1.

However if you think it would be nice to keep these tests around for the future, feel free to leave them but with a `t.skip()` at the top and explanation of why.